src/pharmpy/modeling/data.py

```diff
@@ -205,3 +206,29 @@ def get_observations(model):
 
     """
     return model.dataset.pharmpy.observations
+
+
+def get_mdv(model):
+    """Get MDVs from dataset
+
+    Parameters
+    ----------
+    model : Model
+        Pharmpy model
+
+    Returns
+    -------
+    pd.Series
+        MDVs
+    """
+    try:
+        label = model.dataset.pharmpy.labels_by_type[ColumnType.EVENT]
+    except KeyError:
+        try:
+            label = model.dataset.pharmpy.labels_by_type[ColumnType.DOSE]
+        except KeyError:
```

Comment:
If no event or dose column. Set all rows to MDV=0 (all rows are observations by default)